packages/playwright-test/src/worker.ts

```diff
@@ -88,6 +87,8 @@ process.on('message', async message => {
   }
 });
 
+sendMessageToParent('ready');
```

Comment:
I don't think this changes anything, cross-process communication is async.

Reply:
Reverted in https://github.com/microsoft/playwright/pull/13270/commits/39a0a52c33b816018056ee12f387e3e99674ffa4